README.md

```diff
@@ -133,7 +133,88 @@ Be careful; while you can inspect and change even more as root, Bottlerocket's f
 Rather than a package manager that updates individual pieces of software, Bottlerocket downloads a full filesystem image and reboots into it.
 It can automatically roll back if boot failures occur, and workload failures can trigger manual rollbacks.
 
-Currently, you can update using a CLI tool, updog.
+The update process uses images secured by [TUF](https://theupdateframework.github.io/).
+For more details, see the [update system documentation](sources/updater/).
+
+### Update methods
+
+There are several ways of updating your Bottlerocket hosts:
+
+#### Update API
+
+The [Bottlerocket API](#api) allows you to update and reboot your host with simple API calls.  You can change [settings](#updates-settings) to control which updates are selected.
+
+In general, the update process looks like the following:
+
+![Update API overview](sources/api/Update%20API.png)
```

Comment:
Can we change the space in the name to a hyphen or something?  Maybe lowercase them?  Sorry!